test/testcafe/spec/EnrollAuthenticatorOktaVerify_spec.js

```diff
@@ -250,7 +255,8 @@ const sameDeviceInstruction1 = 'If you don’t have Okta Verify installed, downl
 const sameDeviceInstruction2 = 'Open Okta Verify and follow the steps to add your account.';
 const sameDeviceInstruction3 = 'When prompted, choose Sign In, then enter the sign-in URL:';
 const deviceBootstrapSubtitle = 'To set up Okta Verify on additional devices, you can copy an existing Okta Verify account onto a new device.';
-const deviceBootstrapInstruction1 = 'Open Okta Verify on any of your other Okta Verify devices';
+const deviceBootstrapInstruction1 = 'Open Okta Verify on any of your other Okta Verify devices (Such as your testDevice1).';
+const deviceBootstrapInstructionNoDeviceName = 'Open Okta Verify and follow the steps to add your account.';
```

Comment:
Can remove this?

Reply:
sure: https://github.com/okta/okta-signin-widget/pull/3362/commits/723ebb39c37cd6817a2bf9897472d06492f3beaa